src/vue/components/inflections-table-wide.vue

```diff
@@ -191,6 +193,11 @@ export default {
       }
     },
     collapse: function () {
+      let scrollPos
```

Comment:
why is this being done on collapse? How does that fix the problem reported, which was a failure to scroll to the open table?

Reply:
It is easy.
The problem with scroll is the following:
  - a user clicks on the wide table title
  - it renders the table 
  - at the same time IOS scrolls to the top (I think because of re-rendering)
  finally we need to scroll back

In my fix - before we start render the table - I save the current scroll position.
Then it scrolls up by IOS rules, and I scroll it back (if it has scroll amount).

But on the phone a user doesn't see any scroll at all.

Reply:
ok, well if it works, I'm okay with it, but maybe you could add some comments to the code to explain and remind us why we are doing this?